app/controllers/completed_applications_controller.rb

```diff
@@ -30,10 +30,7 @@ def applications_from_datastore
       status: status_filter, **pagination_params
     ).call
 
-    @paginator = InfinitePaginationV2.new(
```

Comment:
I think there are some locales we can also remove that were used for this infinite paginator.